command/top.go

```diff
@@ -10,26 +11,55 @@ import (
 )
 
 const (
-	topHelp = "Displays the most used distro, terminal, etc."
+	topHelp  = "Displays the most used distro, terminal, etc."
+	topUsage = "`!top` or `!top [Distro OR DeWm OR Terminal etc.]`"
```

Comment:
```suggestion
	topUsage = "!top [Distro OR DeWm OR Terminal etc.]"
```
Parameters are already implied as optional because of `[]`